app/assets/src/components/views/compare/SamplesHeatmapView/SamplesHeatmapControls.jsx

```diff
@@ -218,6 +218,31 @@ export default class SamplesHeatmapControls extends React.Component {
     );
   }
 
+  onSortTaxaChange = order => {
+    if (order === this.props.selectedOptions.taxaSortType) {
```

Comment:
Hmmm....  it's named that way to go with the UI label "Sort taxa by". `currentSortType` wouldn't work because there is also a `sampleSortType` in `selectedOptions`.

Reply:
Sorry, I meant rename `order` -> `currentSortType`

Reply:
ok good point